router/types/types.go

```diff
@@ -5,6 +5,22 @@ import (
 	"time"
 )
 
+// RouteCert describes a TLS certificate for one or more routes
+type RouteCert struct {
+	// ID is the unique ID of this RouteCert
+	ID string `json:"id,omitempty"`
+	// Routes contains the IDs of routes assigned to this cert
+	Routes []string `json:"routes,omitempty"`
+	// TLSCert is the optional TLS public certificate. It is only used for HTTP routes.
+	TLSCert string `json:"tls_cert,omitempty"`
+	// TLSCert is the optional TLS private key. It is only used for HTTP routes.
+	TLSKey string `json:"tls_key,omitempty"`
```

Comment:
I think `TLS` is redundant, remove it from the field names everywhere (just `cert`/`key`).